modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-api/src/main/resources/content/Language.properties

```diff
@@ -1,55 +1,57 @@
-network-bind-host-help=This configuration only applies to EMBEDDED mode. Specify the network interface(s) a node should bind to in order to listen for incoming requests.
+network-bind-host-help=This configuration only applies to EMBEDDED (Development) mode. Specify the network interface(s) a node should bind to in order to listen for incoming requests.
 network-host=Network Host
 network-host-addresses=Network Host Addresses
 network-host-addresses-help=Set the remote HTTP hosts to connect to.
-network-host-help=This configuration only applies to EMBEDDED mode. Set the hostname or IP address bound to the node, which is used to publish (advertise) the host to other nodes in the cluster. This is a shortcut which sets the bind host and the publish host at the same time.
+network-host-help=This configuration only applies to EMBEDDED (Development) mode. Set the hostname or IP address bound to the node, which is used to publish (advertise) the host to other nodes in the cluster. This is a shortcut which sets the bind host and the publish host at the same time.
 network-publish-host=Network Publish Host
-network-publish-host-help=This configuration only applies to EMBEDDED mode. Set the single interface the node advertises to other nodes in the cluster, so that those nodes can connect to it.
+network-publish-host-help=This configuration only applies to EMBEDDED (Development) mode. Set the single interface the node advertises to other nodes in the cluster, so that those nodes can connect to it.
 node-name=Node Name
-node-name-help=This configuration only applies to EMBEDDED mode. Set the name of the node to start. If unset, "liferay" is used.
+node-name-help=This configuration only applies to EMBEDDED (Development) mode. Set the name of the node to start. If unset, "liferay" is used.
 number-of-index-replicas=Number of Index Replicas
 number-of-index-replicas-help=Set the number of replicas for each Liferay index. If unset, no replicas are used. Changes to this value take effect after a full reindex.
 number-of-index-shards=Number of Index Shards
 number-of-index-shards-help=Set the number of shards to use when a Liferay index is created. If unset, a single shard will be used. Changes to this value take effect after a full reindex.
 operation-mode=Operation Mode
-operation-mode-help=The REMOTE operation mode connects to a remote standalone Elasticsearch cluster. The EMBEDDED operation mode starts Portal with an internal Elasticsearch instance in the same JVM. The EMBEDDED operation mode is unsupported.
+operation-mode-help=This configuration is deprecated. The REMOTE (Production) operation mode connects to a remote standalone Elasticsearch cluster, which can also be achieved by enabling Production mode. The EMBEDDED (Development) operation mode starts a child process running an external sidecar Elasticsearch instance. The EMBEDDED (Development) operation mode is unsupported. If Production Mode is enabled, this setting will be ignored.
```

Comment:
Mostly a little re-organization to note that production mode overrules it.

> operation-mode-help=This configuration is deprecated. If Production Mode is enabled, this setting will be ignored. The REMOTE (Production) operation mode connects to a remote standalone Elasticsearch cluster, which can also be achieved by enabling Production mode. The EMBEDDED (Development) operation mode starts a child process running an external sidecar Elasticsearch instance. EMBEDDED operation mode is unsupported.

Reply:
looks good, thanks